Xcode/Sources/HttpServerIO.swift

```diff
@@ -8,7 +8,7 @@
 import Foundation
 import Dispatch
 
-public protocol HttpServerIODelegate: class {
+public protocol HttpServerIODelegate: AnyObject {
```

Comment:
This is a necessary change, but is unrelated to the renaming of the example directory. Could you please remove this from the PR?

Reply:
removed 👍